plugins/orchestrator/docs/rbac-policy.csv

```diff
@@ -1,11 +1,15 @@
 p, role:default/workflowViewer, orchestrator.workflowInstances.read, read, allow
```

Comment:
I'm trying to understand how the policies will be used:
I think the most simple use case is that admins can view and run all workflows
And a group, let's say "developers-a" the admins will want to restrict to use workflow a
And another group, let's say "developers-b" the admins will want to restrict to use workflow b
How will this be done?

Reply:
in this case the policy should have the exact name of the workflow, so there will be no match for any other(*):

```
p, role:default/groupA, orchestrator.workflows.workflow-a, read, allow

p, role:default/groupB, orchestrator.workflows.workflow-b, read, allow
```

The regexp match will not match other workflows names againt this policy, hence it will be denied 

 `* One thing to look out for is that since we're using a regexp match on the name of the workflow then a workflow with the name 'workflow-abc' will also be allowed.`